src/test/java/school/hei/haapi/integration/StudentIT.java

```diff
@@ -178,11 +180,17 @@ void teacher_read_ok() throws ApiException {
     UsersApi api = new UsersApi(teacher1Client);
     Student actualStudent1 = api.getStudentById(STUDENT1_ID);
 
-    List<Student> actualStudents = api.getStudents(1, 20, null, null, null);
+    List<Student> actualStudents = api.getStudents(1, 20, null, null, null, null);
 
     assertEquals(student1(), actualStudent1);
     assertTrue(actualStudents.contains(student1()));
     assertTrue(actualStudents.contains(student2()));
+
+    List<Student> actualStudents2 = api.getStudents(1, 10, null, null, null, COURSE1_ID);
```

Comment:
Put this under the line 183.

The assertions should always be together, as the subject.